README.md

```diff
@@ -4,32 +4,22 @@
   </a>
 </h1>
 
-# Happy Thoughts
```

Comment:
I couldn't open the deployed link on netlify so can't test the functionality

Reply:
It is wierd because I just tested it on my mobile and it worked fine :(